### PR TITLE
Fix/web mode readme

### PR DESCRIPTION
**Pull request by vuvoth**

### CODE REVIEW

File: packages/web/README.md [hunk 1]
```diff
@@ -112,14 +114,15 @@ error("still goes to stderr!");
 debug.log = console.info.bind(console);
 error("now goes to stdout via console.info");
 log("still goes to stdout, but via console.info now");
```
Comment (sczembor) [inline packages/web/README.md:116]: Something is wrong here

File: packages/web/README.md [hunk 2]
```diff
@@ -62,6 +63,7 @@ Now you can use it to execute a transaction:
       payFlags: {},
     };
     let response = await wcSession.executeTx(txParams);
+```
 
 This code will make the transaction, let the user sign it using wallet selected and send it to the network.
```
Comment (sczembor) [inline packages/web/README.md:68]:
```suggestion
This code will create the transaction, let the user sign it using selected wallet and send it to the network.
```

**Review by sczembor — APPROVED** [review]
Pre-approval. Left few suggestions

File: packages/web/README.md (hunk 1 above)
Comment (vuvoth) [reply packages/web/README.md:116]: image
Could you explain more? Please ~~

Comment (sczembor) [reply packages/web/README.md:116]: Oh my bad, thought it was a typo

**Pull request merged**